portal/server/helpers/constructPayload.js

```diff
@@ -25,13 +25,11 @@ const constructPayload = (body, properties, csrf = true) => {
   // Property insertion
   properties
     .filter((property) => property in body)
-    .map((property) => {
+    .forEach((property) => {
```

Comment:
Is `forEach` expensive on execution isn't?

Reply:
happy for us to keep `.map`

Reply:
From what I've read foreach is a bit faster than map in this kind of use case but happy to revert